### DISCUSSION TITLE
How to connect to Coolify-deployed DB w/ Coolify hosted on LAN server

### DISCUSSION THREAD

**Discussion by alexcochran** [coollabsio/coolify#1121] [Q&A]
I have Coolify hosted on a small home server on my local network. When I deploy databases via Coolify, I can't seem to figure out how to connect to them from PCs on the same network. Do I have to connect from the server itself?

**Comment by peaklabs-dev** [comment]
That will not be as simple because the Database is inside a container and has the container network. You could make it public to connect to it.
